### PR TITLE
Improved error message for missing Kotlin classes.

### PR DESCRIPTION
**Pull request by lhstrh**

### CODE REVIEW

File: org.lflang/src/org/lflang/generator/LFGenerator.java [hunk 1]
```diff
@@ -128,12 +128,12 @@ private GeneratorBase createKotlinBaseGenerator(Target target, FileConfig fileCo
...
-                    + "This is likely because you are running the RCA from"
-                    + "Eclipse. The " + target + " code generator is written in Kotlin"
-                    + "and, unfortunately, the Eclipse Kotlin plugin is "
-                    + "broken, preventing us from loading the generator"
-                    + "properly. Please consider building the RCA via Maven.\n"
-                    + "For instructions building RCA via Maven, see: "
+                    + "This is likely because you built Epoch using "
+                    + "Eclipse. The " + target + " code generator is written in Kotlin "
+                    + "and, unfortunately, the plugin that Eclipse uses "
+                    + "for compiling Kotlin code, is broken. "
```
Comment (edwardalee) [inline org.lflang/src/org/lflang/generator/LFGenerator.java:134]: Comma is not needed here.

**Review by edwardalee — APPROVED** [review]
Big improvement, thanks.

File: org.lflang/src/org/lflang/generator/LFGenerator.java (hunk 1 above)
Comment (lhstrh) [reply org.lflang/src/org/lflang/generator/LFGenerator.java:134]: Where? Multiple comma's there :-)

Comment (edwardalee) [reply org.lflang/src/org/lflang/generator/LFGenerator.java:134]: Sorry... I thought the comment would attach to a specific line.  The comma before "is broken"  is wrong.

Comment (lhstrh) [inline org.lflang/src/org/lflang/generator/LFGenerator.java:134]:
```suggestion
                    + "for compiling Kotlin code is broken. "
```

**Review by lhstrh — COMMENTED** [review]
Suggested fix.

**Pull request merged**